packages/apps-config/src/api/spec/index.ts

```diff
@@ -220,7 +222,7 @@ const spec: Record<string, OverrideBundleDefinition> = {
   vodka,
   'web3games-node': web3games,
   'zcloak-network': zCloak,
-  zeitgeist: zeitgeist
+  zeitgeist: zeitgeist,
```

Comment:
```suggestion
  zeitgeist: zeitgeist
```

`yarn lint` will fix all the issues (or most of them), locally.

Reply:
Thank you, fixed